README.md

```diff
@@ -28,4 +28,12 @@ Usage is pretty straight forward, and all the functionnalities are easily shown
 from mpyg321.mpyg321 import MPyg321Player()
```

Comment:
Since we are modifying this file, I made a mistake here: we should remove the parenthesis at the end of the line :)

Reply:
I'm not sure what you mean by this. Which parenthesis do you want to remove?